src/TheGuardianIn2020Banner/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { Button, buttonReaderRevenueBrandAlt, LinkButton } from '@guardian/src-button';
+import { SvgCross, SvgInfo } from '@guardian/src-icons';
+import { OphanComponentEvent } from '@guardian/types/ophan';
+
+import { BrazeClickHandler } from '../utils/tracking';
+import { styles as commonStyles } from '../styles/bannerCommon';
+import { styles } from './styles';
+
+export type Props = {
+    logButtonClickWithBraze: BrazeClickHandler;
+    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
+    ophanComponentId?: string;
+    brazeMessageProps: {
+        header?: string;
+        body?: string;
+    };
+};
+
+const catchAndLogErrors = (description: string, fn: () => void): void => {
+    try {
+        fn();
+    } catch (e) {
+        console.log(`Error (${description}): `, e.message);
+    }
+};
+
+export const COMPONENT_NAME = 'TheGuardianIn2020Banner';
+const THE_GU_IN_2020_URL =
+    'https://www.theguardian.com/info/ng-interactive/2019/dec/19/working-report';
```

Comment:
TODO: This is the URL for the previous year. Update this when we've got the new URL.

Reply:
Also reminding: We should double check it has the appropriate CMP params or other tracking on it.